test/runtests.jl

```diff
@@ -382,5 +382,13 @@ include("setup.jl")
     end
 end
 
+@testset "ftp download" begin
```

Comment:
```suggestion
@__MODULE__ == Main && @testset "ftp download" begin
```
This will not run this test suite during Julia CI (which generates an anonymous module that it runs each test suite in). It's fine to use a random FTP server for CI for this package by itself but Julia CI runs so much that it will surely cause problems and flakiness.